modules/enterprise/comm/src/main/java/org/rhq/enterprise/communications/ServiceContainer.java

```diff
@@ -1246,6 +1246,15 @@ private void setupServerConnector() throws Exception {
             if (transport.equals("https")) {
                 connector_config.put("SSLImplementation", RemotingSSLImplementation.class.getName());
             }
+
+            if (transport.equals("sslservlet") || transport.equals("https")) {
+                String tomcatAuthMode = "rhq.server.tomcat.security.client-auth-mode";
+                String tomcatAuthModeValue = System.getProperty("rhq.server.tomcat.security.client-auth-mode");
+                if (!tomcatAuthModeValue.equals("true") && !tomcatAuthModeValue.equals("false")) {
```

Comment:
This value is passed directly to jboss-as.
Couldn't find much info but it seems that these values are in lower case.
(From the docs): `When using sslservlet set rhq.server.tomcat.security.client-auth-mode parameter to true`
[BZ-1369081](https://bugzilla.redhat.com/show_bug.cgi?id=1369081#c5)

and found [this](https://github.com/wildfly/wildfly/blob/6b61a6003f704221f66dcd9f418bcb7af88fb9ab/legacy/web/src/main/java/org/jboss/as/web/WebMigrateOperation.java#L1042-L1065)